pricing-page-exercise/onur/index.html

```diff
@@ -4,10 +4,120 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Font Google -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap" rel="stylesheet">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
   <link rel="stylesheet" href="./styles.css">
   <title>Pricing page Exercise</title>
 </head>
 <body>
-  
+  <header>
+    <nav class="navbar">
+        <div><h3 class="heading-small">Company name</h3></div>
+        <ul>  
+            <li><a class="navlink" href="#Features">Features</a></li>
+            <li><a class="navlink" href="#Enterprise">Enterprise</a></li>
+            <li><a class="navlink" href="#Support">Support</a></li>
+            <li><a class="navlink" href="#Pricing">Pricing</a></li>
+            <li><a class="btn" href="#">Sign Up</a></li>
+        </ul>  
+    </nav>
+ </header>
+ <main>
```

Comment:
The opening <main> tag doesn't have a corresponding closing </main> tag. Make sure to add the closing tag right before the opening <footer> tag.